CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 * [#6855](https://github.com/rubocop-hq/rubocop/pull/6855): Fix an exception in `Rails/RedundantReceiverInWithOptions` when the body is empty. ([@ericsullivan][])
+* [#6856](https://github.com/rubocop-hq/rubocop/pull/6856): Fix AutoCorrect exception in `Style::BlockComments` when the file is missing a trailing blank line. ([@ericsullivan][])
```

Comment:
It is a small point. It would be better to write `Style::BlockComments` as `Style/BlockComments`.

Reply:
ah, good point. The wording was a little different from previous auto-correction fixes, so I adjusted that as well.